GWAS/liftover.ipynb

```diff
@@ -251,13 +251,17 @@
     "output: f'{cwd}/{output_file}'\n",
     "python: input = f'{cwd:a}/utils.py', expand = '${ }', stderr = f'{_output[0]:n}.stderr', stdout = f'{_output[0]:n}.stdout'\n",
     "    \n",
+    "    \n",
+    "    import os.path\n",
     "    input_path=${_input[0]:r}\n",
     "    output_path=${_output[0]:r}\n",
-    "    fr = '${fr}'\n",
-    "    to = '${to}'\n",
+    "    fr = f'${fr}'\n",
+    "    to = f'${to}'\n",
     "    remove_missing=${remove_missing}\n",
     "    rename = ${rename}\n",
-    "    yml_file = '${yml_file}'\n",
+    "    yml_file = f'${yml_file}'\n",
```

Comment:
you can try this: 

```
import pathlib
yml_file  = pathlib.Path(${yml_file:r})
```
this allows you to use `is_file()` later. of course you can do it with other approaches.